src/veins/modules/mobility/traci/TraCIScenarioManager.h

```diff
@@ -100,7 +100,18 @@ class VEINS_API TraCIScenarioManager : public cSimpleModule {
         return hosts;
     }
 
+    /**
+     * Predicate indicating a successful connection to the TraCI server.
+     *
+     * \note Once the connection has been established, this will return true even when the connection has been torn down again.
```

Comment:
I believe we're using @note in other places. Might be nice to stay consistent here.